checker/src/org/checkerframework/checker/lock/LockAnnotatedTypeFactory.java

```diff
@@ -23,9 +44,32 @@
 import java.util.Set;
 
 import javax.lang.model.element.AnnotationMirror;
+import javax.lang.model.element.Element;
+import javax.lang.model.element.ElementKind;
+import javax.lang.model.element.ExecutableElement;
 import javax.lang.model.element.VariableElement;
 
-import com.sun.source.tree.Tree;
+// TODO: Here is a list of nice-to-have features/tests but not critical to release the Lock Checker:
```

Comment:
I would move this list to the Lock Checker class. (Though I'm not sure this list belongs in code at all.)